lib/floe.rb

```diff
@@ -45,7 +45,27 @@ def self.logger
     @logger ||= NullLogger.new
   end
 
+  # Set the logger to use
+  #
+  # example
+  #
+  #   require "logger"
+  #   Floe.logger = Logger.new($stdout)
+  # @param logger [Logger] logger to use for debugging
```

Comment:
```suggestion
  # @example
  #   require "logger"
  #   Floe.logger = Logger.new($stdout)
  #
  # @param logger [Logger] logger to use for debugging
```

Reply:
I'm not sure we should say "for debugging", it's also just for regular logging of actions.